src/index.tsx

```diff
@@ -1,34 +1,38 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import 'index.css';
-import App from 'components/App/App';
-import {store} from 'store/store';
-import {Provider} from 'react-redux';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "index.css";
+import App from "components/App/App";
+import { store } from "store/store";
+import { Provider } from "react-redux";
+import * as serviceWorker from "./serviceWorker";
 import CssBaseline from "@material-ui/core/CssBaseline";
-import axios from 'axios';
+import axios from "axios";
 
 axios.interceptors.response.use(
-  response => {
+  (response) => {
     return response;
   },
-  function(error) {
+  function (error) {
     if (error?.response?.status === 400) {
       alert(error.response.data?.data);
     }
 
+    if (error?.response?.status === 401 || error?.response?.status === 403) {
+      alert(error.response.data?.message);
```

Comment:
Alerting is not working. CORS issue.

Reply:
> Alerting is not working. CORS issue.

fixed some days ago 
![image](https://user-images.githubusercontent.com/8881978/121171545-4c26ed80-c85f-11eb-9f86-f604ae762c19.png)